src/Routes.js

```diff
@@ -21,25 +22,19 @@ export const Routes = () => {
             ...mergeWithEntities(tableReducer)
         });
     }, [getRegistry]);
-    const searchParams = new URLSearchParams(location.search);
 
     return (
         <Suspense fallback="">
             <Switch>
                 <Route
                     exact
                     path={routes.table}
-                    render={() => <InventoryTable
-                        status={searchParams.getAll('status')}
-                        source={searchParams.getAll('source')}
-                        filterbyName={searchParams.getAll('hostname_or_id')}
-                        tagsFilter={searchParams.getAll('tags')?.[0]?.split?.(',').reduce?.(tagsMapper, [])}
-                        operatingSystem={searchParams.getAll('operating_system')}
-                        rhcdFilter={searchParams.getAll(RHCD_FILTER_KEY)}
-                        updateMethodFilter={searchParams.getAll(UPDATE_METHOD_KEY)}
-                        page={searchParams.getAll('page')}
-                        perPage={searchParams.getAll('per_page')}
-                    />}
+                    render={() =>
+                        <RenderWrapper
+                            cmp={InventoryTable}
+                            isRbacEnabled
+                            {...getSearchParams()}
```

Comment:
It might make sense to wrap this in a `useMemo`.

Reply:
Good point!